spring-core/src/main/java/dao/EventDao.java

```diff
@@ -0,0 +1,74 @@
+package dao;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import model.Event;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public final class EventDao {
+
+    private String filePath;
+    private Map<Long, Event> eventStorage;
+
+    private void init() throws IOException {
+        Resource resource = new ClassPathResource(filePath);
+        if (resource.isFile()) {
+            String users = new String(Files.readAllBytes(resource.getFile().toPath()));
+            this.eventStorage = new ObjectMapper().readValue(users, new TypeReference<Map<Long, Event>>() {
+            });
+        }
+    }
+
+    public void setFilePath(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public Event findEventById(long eventId) {
+        return eventStorage.get(eventId);
+    }
+
+    public List<Event> findEventByTitle(String title) {
+        return eventStorage.values().stream().filter(event -> event.getTitle().equals(title)).collect(Collectors.toList());
+    }
+
+    public List<Event> findEventByDay(Date date) {
+        return eventStorage.values().stream().filter(event -> {
+            Calendar calendar1 = Calendar.getInstance();
```

Comment:
celendar1 celendar2 is not clear naming for variables, hard to read this code with this naming